web_access_rule_buttons/models.py

```diff
@@ -19,6 +19,12 @@ def check_access_rule_all(self, operations=None):
         operations = ['read', 'create', 'write', 'unlink']
     result = {}
     for operation in operations:
+        if self.is_transient() and not self.ids:
```

Comment:
I'm pretty sure transient model records have an ID, the only thing is that they get deleted by a cron job. Are you sure `self.is_transient()` is not enough?